### PR TITLE
Controllable ambient light color

### PR DESCRIPTION
**Pull request by no1hitjam**
I added an AmbientLight resource that controls the ambient light color in the current PBR fragment shader. Hopefully I'm not stepping on the toes of the active work being done on the real PBR shader.

### CODE REVIEW

File: crates/bevy_pbr/src/render_graph/lights_node.rs [hunk 1]
```diff
@@ -75,19 +76,34 @@ pub struct LightsNodeSystemState {
     max_lights: usize,
 }
 
+// Ambient color.
+#[derive(Debug)]
+pub struct AmbientLight {
```
Comment (cart) [inline crates/bevy_pbr/src/render_graph/lights_node.rs:81]: Can we move this to `light.rs` and make sure its exported at `bevy_pbr::AmbientLight`?

**Review by cart — COMMENTED** [review]
I think this is nice interim solution while we wait for a more advanced lighting system. Thanks

File: crates/bevy_pbr/src/render_graph/lights_node.rs (hunk 1 above)
Comment (no1hitjam) [reply crates/bevy_pbr/src/render_graph/lights_node.rs]: Done

Comment (no1hitjam) [reply crates/bevy_pbr/src/render_graph/lights_node.rs]: Wait, I missed the pub(crate).

Comment (no1hitjam) [reply crates/bevy_pbr/src/render_graph/lights_node.rs]: Something weird is happening where my latest commit isn't being added here. I think I may create a new PR.

Comment (cart) [reply crates/bevy_pbr/src/render_graph/lights_node.rs]: It all gets squashed down in the end, so thats not a problem!

**Comment by DJMcNab** [comment]
@no1hitjam please respond in #2373 for the relicense to MIT/Apache 2.0. Thanks!

**Pull request merged**